ibis-server/app/model/metadata/redshift.py

```
@@ -0,0 +1,171 @@
+                ON ccu.constraint_name = tc.constraint_name
+            WHERE tc.constraint_type = 'FOREIGN KEY'
+            """
+        # we reuse the connector.query method to execute the SQL
+        # so we have to give a limit to avoid too many rows
+        # the default limit for constraints metadata is 500, i think it's a sensible limit
+        response = self.connector.query(sql, limit=500).to_dict(orient="records")
+        constraints = []
+        for row in response:
+            constraints.append(
+                Constraint(
+                    constraintName=self._format_constraint_name(
```

Pylint (3.3.7)

[error] Instance of 'RedshiftMetadata' has no '_format_constraint_name' member

(E1101)